### ISSUE TITLE
Bulk update log text option for Drafts and Pending Logs

### ISSUE THREAD

**Issue by cachly** [cachly/issues-tracker#4339]
**[Original report](https://bitbucket.org/zedsaidstudio/cachly/issues/4338) by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

**Comment by cachly** [comment]
**Original comment by Zed Said Studio LLC (Bitbucket: [zedsaid](https://bitbucket.org/zedsaid), GitHub: [zedsaid](https://github.com/zedsaid)).**

----------------------------------------

> I know that we had the bulk logging feature before but now with the new drafts I wish it even more. I make very often hiking or biking trails with about 50 geocaches. Since each individual cache is not really interesting I can’t write an interesting log for each. So I would like to write for all 50 a nice long log that tells my day / trip. I currently do it this way. But I have to currently always open each offline log -> open the log screen -> insert the log -> go back 2 times etc.. This is of course very tedious. It would be great if I could select all 50 offline logs / drafts and then a log screen would open. If I press save afterwards, this log text is inserted in all 50 offline logs / drafts. It would just save so much time . I know that there are text templates, but I can’t use them for this, because I don’t know my log text in advance.
>
>   
> I know that bulk logging is very controversial, but I think this feature would not change anything, because bulk logging is already possible with text templates.  
> I think the feature would also be a unique selling point, as I don’t know of any other iOS app that could do that.

[https://cachly.slack.com/archives/CJY3A4APM/p1643870613783979](https://cachly.slack.com/archives/CJY3A4APM/p1643870613783979)

**Comment by Kyzabra** [comment]
This looks like
The function I'm chasing. The ability to flag a cache as found during the day, then compose a log and apply to all found would be amazing. I'm sure that's what we did back in geosphere days.

**Comment by JanLankhorst** [comment]
Yes! This is exactly what I was searching for in Cachly. My habits started in the good old days of GSAK. When I arrived home  I would compose the log text for the entire day and apply that template to the logs. Great that community wishes are implemented.

**Comment by cachly** [comment]
This is planned for Cachly 8.3.
